misc/snapshotter/Dockerfile

```diff
@@ -1,12 +1,21 @@
 FROM alpine:3.17.0 AS sourcer
 
+ARG TARGETARCH
+
 RUN apk add --no-cache curl
 RUN apk add --no-cache --upgrade grep
-RUN export NYDUS_VERSION=$(curl --silent "https://api.github.com/repos/dragonflyoss/image-service/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")') && \
-    wget https://github.com/dragonflyoss/image-service/releases/download/$NYDUS_VERSION/nydus-static-$NYDUS_VERSION-linux-amd64.tgz && \
+RUN unset NYDUS_VERSION && \
+    while [ -z $NYDUS_VERSION ]; do \
+        export NYDUS_VERSION=$(curl --silent "https://api.github.com/repos/dragonflyoss/image-service/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")'); sleep 1; \
```

Comment:
It's better to limit the total retry times.

Reply:
Done.

Reply:
It still failed when i retry 20 times. Maybe we could assume it will work eventually.

Reply:
It works now. 10 minutes are enough.